app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -484,6 +506,50 @@ MinionPoller = {
   }
 };
 
+function removeNode(minionId, hostname) {
+  var $alert = $('.failed-removal-alert');
+  var error = 'Failed to remove node '+ minionId +'. Please try again.';
+
+  if (confirm('Are you sure you want to remove ' + hostname + '?')) {
+    setPendingRemoval(minionId);
+    $.ajax({
+      url: '/minions/'+ minionId +'/remove-minion.json',
+      method: 'POST',
+      data: { minion_id: minionId },
+      dataType: 'text'
+    }).done(function(){
+      removePendingRemoval();
```

Comment:
By calling this here you'll re-enable things for a few seconds until the next fetch happens, which is an undesired behavior. Leftover while testing?

Reply:
Noticed that behavior didn't know what was doing it, thanks.